### PR TITLE
Add cypress test

### PR DESCRIPTION
**Pull request by MorganLove**
**Description**

Quick try-out of cypress

**Testing**

- Creating a new audit

**Progress**

- Just a proof-of-concept, happy to merge it in

### CODE REVIEW

File: client/cypress/end-to-end/audit-setup.spec.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+describe('Audit setup', () => {
+  it('Creates an audit', () => {
+    cy.visit('/')
+    cy.loginAuditAdmin('audit-admin-cypress@example.com')
+    cy.contains('Give your new audit a unique name.').type('Audit One')
```
Comment (jonahkagan) [inline client/cypress/end-to-end/audit-setup.spec.js:5]: you can use testing library commands like `findByRole` cuz I installed @testing-library/cypress: https://testing-library.com/docs/cypress-testing-library/intro

**Comment by MorganLove** [comment]
> nice, glad it works. any particular reason to create a new spec rather than just start fleshing out the spec I already made?

Not particularly, other than in case you wanted to tweak it I didn't want to step on your code's toes and cause merge conflicts, haha. I figured this one would end up being larger and focused specifically on audit creation, and the other one would be smaller and more basic, but I could have assumed wrong. :slightly_smiling_face:

**Comment by jonahkagan** [comment]
Ah gotcha. I meant mine to be just the bare minimum test for you to build on :) I wasn't planning on adding to it. In fact I already merged it

**Comment by MorganLove** [comment]
Rebasing this turned into a mess, so I'm just creating a new PR with the test on the original spec.

**Pull request closed without merging**